ios/RNSBarButtonItem.mm

```diff
@@ -20,25 +20,32 @@ - (instancetype)initWithConfig:(NSDictionary<NSString *, id> *)dict
     return self;
   }
 
-  self.title = dict[@"title"];
```

Comment:
Why did you resign from setting title here and postponed it for later? Was there a particular reason here?

Reply:
But shouldn't be needed anymore since from what I understand also remote images will be loaded synchronously in production mode?